core/src/smartcontracts/isi/triggers.rs

```diff
@@ -73,10 +80,16 @@ pub mod isi {
             _authority: <Account as Identifiable>::Id,
             wsv: &WorldStateView<W>,
         ) -> Result<(), Self::Error> {
-            let trigger = self.object_id.clone();
+            let trigger_id = self.object_id.clone();
             wsv.modify_triggers(|triggers| {
-                triggers.remove(&trigger)?;
```

Comment:
Yeah, it was a *Repetition* error before, but it was done in `TriggerSet`. Now it is done here, because `TriggerSet` should know nothing about instructions